BrainPortal/app/controllers/nh_messages_controller.rb

```diff
@@ -120,6 +112,6 @@ def destroy #:nodoc:
   private
 
   def message_params
-    params.require(:message).permit(:header, :description, :variable_text, :message_type, :read, :user_id, :expiry, :last_sent, :critical, :display, :send_email, :group_id, :sender_id)
+    params.require(:message).permit(:header, :description, :variable_textent, :group_id)
```

Comment:
Did you join 2 attributes? `variable_textent`

Reply:
a typo